docs/configuring-playbook-synapse-admin.md

```diff
@@ -6,7 +6,6 @@ It's a web UI tool you can use to **administrate users and rooms on your Matrix
 
 See the project's [documentation](https://github.com/Awesome-Technologies/synapse-admin) to learn what it does and why it might be useful to you.
 
-
```

Comment:
Can you restore this and the other removed lines?

Reply:
sorry. the lines are restored